backend/plugins/accounting_api/src/modules/accounting/@types/constants.ts

```
@@ -6,13 +6,20 @@ export const ACCOUNT_KINDS = {
+  EXCHANGE_DIFF: 'exchangeDiff',
   INVENTORY: 'inventory',
+  INV_FOLLOW: 'invFollow',
   FIXED_ASSET: 'fixedAsset',
-  TAX: 'tax',
-  ALL: ['main', 'cash', 'bank', 'debt', 'inventory', 'fixedAsset', 'tax'],
+  ALL: [
+    'main', , 'tax',
+    'cash', 'bank', 'debt', 'exchangeDiff',
+    'inventory', 'invFollow',
+    'fixedAsset',
+  ],
```

Biome (2.1.2)

[error] This array contains an empty slots..

The presences of empty slots may cause incorrect information and might be a typo.
Unsafe fix: Replace hole with undefined


(lint/suspicious/noSparseArray)